buildstockbatch/gcp/gcp.py

```diff
@@ -190,8 +190,58 @@ def get_AR_repo_name(gcp_project, region, repo):
         """Returns the full name of a repository in Artifact Registry."""
         return f"projects/{gcp_project}/locations/{region}/repositories/{repo}"
 
+    def check_output_dir(self):
+        """Check for existing results files in the output directory."""
+        if self.missing_only:
+            return
+        storage_client = storage.Client(project=self.gcp_project)
+        output_dir = os.path.join(self.cfg["gcp"]["gcs"]["prefix"], "results", "simulation_output")
+        bucket = storage_client.bucket(self.gcs_bucket)
+        blobs = bucket.list_blobs(prefix=os.path.join(output_dir, "results_job"))
+
+        try:
+            blob = next(blobs)
+        except StopIteration:
+            return
+
+        prefix_for_deletion = self.cfg["gcp"]["gcs"]["prefix"]
+        blobs_for_deletion = bucket.list_blobs(prefix=prefix_for_deletion)
+        user_choice = (
+            input(
+                f"Output files are already present in bucket {self.gcs_bucket}! For example, {blob.name} exists. "
+                f"Do you want to delete all the files in {prefix_for_deletion}? (yes/no): "
```

Comment:
Thinking maybe we add also the warning that if you delete these files, you won't be able to recover them or something like that?

Reply:
Added "permanently" to the prompt. Do you think that's enough?